pkgs/applications/misc/mission-center/default.nix

```
@@ -0,0 +1,124 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, fetchFromGitHub
+, cargo
+, libxml2
+, meson
+, ninja
+, pkg-config
+, python311Full
+, rustPlatform
+, symlinkJoin
+, rustc
+, wrapGAppsHook4
+, appstream-glib
+, blueprint-compiler
+, cairo
+, cmake
+, desktop-file-utils
+, dmidecode
+, gdk-pixbuf
+, gettext
+, glib
+, graphene
+, gtk4
+, libadwaita
+, libdrm
+, mesa
+, pango
+, sqlite
+, udev
+, wayland
+}:
+
+let
+  nvtop = fetchFromGitHub {
+    owner = "Syllo";
+    repo = "nvtop";
+    rev = "9a8458b541a195a0c5cadafb66e240962c852b39";
+    hash = "sha256-iFBZbESRTuwgLSUuHnjcXwmpvdeQrd3oUJd7BRyxu84=";
+  };
+in
+stdenv.mkDerivation rec {
+  pname = "mission-center";
+  version = "0.2.4";
+
+  src = fetchFromGitLab {
+    owner = "mission-center-devs";
+    repo = "mission-center";
+    rev = "v${version}";
+    hash = "sha256-WLSTYpdhOW+Fs5+XP06F8Tgro5nqi8m5zZ77jdUeUpM=";
+  };
+
+  cargoDepsBase = rustPlatform.importCargoLock {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+       "pathfinder_canvas-0.5.0" = "sha256-k2Sj69hWA0UzRfv91aG1TAygVIuOX3gmipcDbuZxxc8=";
+    };
+  };
+  cargoDepsProxy = rustPlatform.importCargoLock {
+    lockFile = ./proxy-Cargo.lock;
+  };
+
+  cargoDeps = symlinkJoin {
+    name = "cargo-vendor-dir";
+    paths = [ cargoDepsBase cargoDepsProxy ];
+  };
+
+
+  nativeBuildInputs = [
+    blueprint-compiler
+    cargo
+    libxml2
+    meson
+    ninja
+    pkg-config
+    python311Full
```

```suggestion
    python311
```
which features do you need from python3Full?

---

only `argparse`, `os`, and `sqlite3`

---

It seems that it works fine with `python311` so I have put that.